### ISSUE TITLE
[docker-compose fail] Error: Host 'portus_web_run_1' is not allowed to connect to this MariaDB server

### ISSUE THREAD

**Issue by SEJeff** [SUSE/Portus#654]
I literally just did:

    ./compose-setup.sh

And it always failed, so I uncommented out the `&>/dev/null` in `setup_database()` and get this:

```ruby
Portus: configuring database...WARNING: Nokogiri was built against LibXML version 2.9.1, but has dynamically loaded 2.8.0
#<Mysql2::Error: Host 'portus_web_run_1' is not allowed to connect to this MariaDB server>
Couldn't create database for {"adapter"=>"mysql2", "encoding"=>"utf8", "host"=>"db", "username"=>"root", "password"=>"portus", "database"=>"portus_development"}, {:charset=>"utf8", :collation=>"utf8_unicode_ci"}
(If you set the charset manually, make sure you have a matching collation)
#<Mysql2::Error: Host 'portus_web_run_1' is not allowed to connect to this MariaDB server>
Couldn't create database for {"adapter"=>"mysql2", "encoding"=>"utf8", "host"=>"db", "username"=>"root", "password"=>"portus", "database"=>"portus_test"}, {:charset=>"utf8", :collation=>"utf8_unicode_ci"}
(If you set the charset manually, make sure you have a matching collation)
-- create_table("activities", {:force=>:cascade})
rake aborted!
Mysql2::Error: Host 'portus_web_run_1' is not allowed to connect to this MariaDB server
/portus/db/schema.rb:16:in `block in <top (required)>'
/portus/db/schema.rb:14:in `<top (required)>'
Tasks: TOP => db:schema:load
(See full trace by running task with --trace)
```

This seems likely to be the culprit:

    #<Mysql2::Error: Host 'portus_web_run_1' is not allowed to connect to this MariaDB server>

And I'm using portus 8ebd8851e75a181 from the latest master.

**Comment by SEJeff** [comment]
Looks like the mariadb folks broke something, this seems to make things better:

```patch
diff --git a/docker-compose.yml b/docker-compose.yml
index 106e4b2..431401d 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -16,7 +16,7 @@ crono:
     - db
 
 db:
-  image: library/mariadb
+  image: library/mariadb:10.0
   environment:
     MYSQL_ROOT_PASSWORD: portus
```

**Comment by mssola** [comment]
I've tried to reproduce this with the current `latest` tag (it's pointing to `10.1.10`), and it's working for me. Maybe you had another `mariadb:latest` on your machine already ?

**Comment by flavio** [comment]
Fixed with PR #649
